src/main/java/core/basesyntax/filework/FileReader.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.filework;
```

Comment:
```suggestion
package core.basesyntax.service;
```
Let’s  name this package service and move other services into this package